primer.md

```diff
@@ -253,7 +253,10 @@ The `id` attribute is meant to be a value that is unique across all events
 related to one event source (where each event source is uniquely identified by
 its CloudEvents `source` attribute value). While the exact value used is
 producer defined, receivers of CloudEvents from a single event source can be
-assured that no two events will share the same `id` value. The only exception to
+assured that no two events will share the same `id` value. (We are implicitly 
```

Comment:
I'd remove the parens and replace the `? Since` with `, since`

Reply:
Makes sense. I put in parenthesis as I was not sure if we should state that in the primer. Although it makes it more clear with your suggestion.